### PR TITLE
Added stat for strife

### PR DESCRIPTION
**Pull request by abelito75**
Adding a quick statistic for Strife
facestat
tooltip

I had been asked for the table to be there so I threw that into the tool tip :)

### CODE REVIEW

File: src/parser/shared/modules/spells/bfa/essences/Strife.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    this.active = this.selectedCombatant.hasEssence(SPELLS.CONFLICT.traitId);
+    if (!this.active) {
+      return;
+    }
+
+    this.vers = calculatePrimaryStat(455, 53, this.selectedCombatant.neck.itemLevel);
+
+
+    this.statTracker.add(SPELLS.STRIFE_BUFF.id, {
+      versatility: this.versGain,
```
Comment (HawkCorrigan) [inline src/parser/shared/modules/spells/bfa/essences/Strife.js]: This looks wrong

File: src/parser/shared/modules/spells/bfa/essences/Strife.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+  array = [];
+  currentStack = 0;
+  vers = 0;
+  lastApplication = 0;
+
+  constructor(...args) {
+    super(...args);
+    this.active = this.selectedCombatant.hasEssence(SPELLS.CONFLICT.traitId);
+    if (!this.active) {
+      return;
```
Comment (HawkCorrigan) [inline src/parser/shared/modules/spells/bfa/essences/Strife.js]: This does nothing.

**Comment by emallson** [comment]
@abelito75 any movement on this? looks like a pretty minor change needed

**Comment by abelito75** [comment]
Just been a bit busy after Christmas I'll get this and the other pr of mine knocked out

**Comment by abelito75** [comment]
I snuck in some comment changes into other files since they were randomly pulled up in vscode and I had the brain power to explain them better

**Pull request merged**